### PR TITLE
replace hrs/wk with hours, do not show start date

### PR DESCRIPTION
**Pull request by sblack-usu**
@crivet - This should have all the course display tweaks.  I wasn't able to put together a test course that displayed the estimated hrs/wk on a course card within the courses page, but the changes should cover it.  I tried creating a course that has a start date before today, but nothing seemed to work.  The estimated effort shows up correct on the course about page, so not sure what is going on.  Please verify the course card displays correctly on your instance if you can before merging.

I'll comment on the new theme files to highlight where the changes occur.

### CODE REVIEW

File: hydrolearn-theme/cms/templates/settings.html [hunk 1]
```diff
@@ -0,0 +1,645 @@
...
+            <div class="group-settings requirements">
+              <header>
+                <h2 class="title-2">${_("Requirements")}</h2>
+                <span class="tip">${_("Expectations of the students taking this course")}</span>
+              </header>
+
+              <ol class="list-input">
+                % if about_page_editable:
+                <li class="field text" id="field-course-effort">
+                  <label for="course-effort">${_("Hours of Effort")}</label>
```
Comment (sblack-usu) [inline hydrolearn-theme/cms/templates/settings.html:552]: was `Hours of Effort per Week`

File: hydrolearn-theme/cms/templates/ux/reference/fragments/course-settings.html [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+
+            <div class="group-settings requirements">
+              <header>
+                <h2 class="title-2">Requirements</h2>
+                <span class="tip">Expectations of the students taking this course</span>
+              </header>
+
+              <ol class="list-input">
+                <li class="field text" id="field-course-effort">
+                  <label for="course-effort">Hours of Effort</label>
```
Comment (sblack-usu) [inline hydrolearn-theme/cms/templates/ux/reference/fragments/course-settings.html:410]: Was `Hours of Effort per Week`

File: hydrolearn-theme/lms/static/sass/multicourse/_courses.scss [hunk 3]
```diff
@@ -104,6 +104,7 @@ $facet-background-color: #007db8;
             .course-date{ 
               grid-area: start_date; 
               padding: 0;
+              display: none;
```
Comment (sblack-usu) [inline hydrolearn-theme/lms/static/sass/multicourse/_courses.scss:107]: this is for the course card on the courses page

File: hydrolearn-theme/lms/templates/courseware/course_about.html [hunk 4]
```diff
@@ -244,43 +244,6 @@ <h1>
           <li class="important-dates-item"><span class="icon fa fa-info-circle" aria-hidden="true" title="Link to create tar.gz file of course"></span><p class="important-dates-item-title">${_("Course Export")}</p><a class="important-dates-item-text course-export" href="${studio_export_url}">Export Link</a></li>
 
           <li class="important-dates-item"><span class="icon fa fa-info-circle" aria-hidden="true"></span><p class="important-dates-item-title">${_("Course Number")}</p><span class="important-dates-item-text course-number">${course.display_number_with_default}</span></li>
```
Comment (sblack-usu) [inline hydrolearn-theme/lms/templates/courseware/course_about.html:246]: this is for the course start date on the course about page

File: hydrolearn-theme/lms/templates/discovery/course_card.underscore [hunk 5]
```diff
@@ -38,7 +38,7 @@
             
             <div class='course-weekly-effort' aria-hidden="true">
                 <% if (typeof(effort) != "undefined") { %>
-                    <%- gettext("Effort:") %> <%- effort %> <%- gettext("hrs/wk") %>
+                    <%- gettext("Effort:") %> <%- effort %> <%- gettext("hours") %>
```
Comment (sblack-usu) [inline hydrolearn-theme/lms/templates/discovery/course_card.underscore:41]: This is the piece I was not able to get a test for.  It'll show on the course card of the courses page

**Review by crivet — APPROVED** [review]
Everything looks good on my end. Built with the updated changes in my instance and everything appears correct. i.e. 'hrs/wk' replaced with 'hours' in expected places, and start date hidden

**Pull request merged**